lib/bloc/backup/backup_bloc.dart

```diff
@@ -331,6 +319,78 @@ class BackupBloc {
         encryptionKey?.key, encryptionKey?.type);
   }
 
+  Future _listSnapshots(ListSnapshots action) async {
+    String backups = await _breezLib.getAvailableBackups().catchError(
```

Comment:
When using this pattern: await  _breezLib.getAvailableBackups().catchError(...) won't it return with success? It seems as the code will continue after the catch to line 343. Or am I wrong?

Reply:
Yes, we're missing return calls after resolving with error.

Reply:
I will be debugging & addressing all error handling issues.